packages/engine/Source/Scene/Scene.js

```diff
@@ -1938,7 +1939,8 @@ Scene.prototype.updateFrameState = function () {
     camera.upWC
   );
   frameState.occluder = getOccluder(this);
-  frameState.minimumTerrainHeight = 0.0;
+  frameState.minimumTerrainHeight =
```

Comment:
I'm curious why you chose to use  `ApproximateTerrainHeights._defaultMinTerrainHeight` over `ApproximateTerrainHeights.getMinimumMaximumHeights`. Is this only to preserve the workaround for non-Earth cases?